src/components/Breadcrumbs/Breadcrumb.styles.js

```diff
@@ -7,11 +7,7 @@ const StyledBreadcrumb = ListUnstyled.extend`
   font-weight: ${typography.weight.semiBold};
   display: flex;
   flex-flow: row;
-  ${({ childrenLen }) =>
-    childrenLen &&
-    `
-    flex: 0 1 ${Math.floor(100 / childrenLen)}%;
```

Comment:
These are styles for Breadcrumbs container. `flex` does not affect it, unless it was inside another flex-container. but it's not.